src/V3EmitCSyms.cpp

```diff
@@ -184,7 +184,15 @@ class EmitCSyms final : EmitCBaseVisitorConst {
         return out;
     }
 
+    /// (scp, m_vpiScopeCandidates, m_scopeNames) -> m_scopeNames
+    /// Look for parent scopes of scp in m_vpiScopeCandidates (separated by __DOT__ or ".")
+    /// Then add/update entry in m_scopeNames if not already there
     void varHierarchyScopes(string scp) {
+
+        // we want no result to be -1, so use ints
+        int dot_pos = scp.rfind('.');
+        int DOT_pos = scp.rfind("__DOT__");
```

Comment:
Rename so only difference isn't case please.
This will get a warning on some formal tools as rfind returns size_t 64 bits and you are dropping the upper bits.  You can have a static_cast to int, but maybe it's cleaner to use standard string::size_type and below only do the compare "if (DOT_pos != std::string::npos && dot_pos != std::string::npos)?

Reply:
I switched it to use long, is fine with you? I think it's worth using signed for simpler comparisons, and we won't be having a scope string > 2^63-1 bits

Reply:
rfind returns size_t.  While I agree we won't really care about the lost bit, assigning directly to long will still make a lint warning.